utils/download_utils.py

```diff
@@ -0,0 +1,82 @@
+import pandas as pd
+import numpy as np
+from shapely.geometry import Point
+from shapely.geometry.polygon import Polygon
+
+
+def parse_outline_data(raw_outline_data: str) -> np.array:
+    """
+    parse outline data extracted with IDR stream into numpy array format
+    Parameters
+    ----------
+    raw_outline_data : str
+        string of outline data
+    Returns
+    -------
+    np.array
+        parsed outline data
+    """
+    outline_data = []
+
+    raw_outline_data = raw_outline_data[1:-1]
```

Comment:
Consider adding comments to this block to describe the important parts about what is happening. With complex string or list formatting, it may otherwise be easy to lose track of why certain steps were taken here.